.vsts-ci.yml

```diff
@@ -35,68 +41,71 @@ jobs:
       artifactType: container
       artifactName: RayTraceBenchmark-Wasm
 
-- job: linux
-  container: nv-bionic-wasm
+- job: Linux
+  container: unoplatform/wasm-build:2.0
 
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-18.04'
 
   variables:
-    NUGET_PACKAGES: $(build.sourcesdirectory)/.nuget
-    EMSCRIPTEN_VERSION: 1.38.48-upstream
+    NUGET_PACKAGES: $(Build.SourcesDirectory)/.nuget
 
   steps:
-  - checkout: self
-    clean: true
-
   - bash: |
-      cd /emsdk
-      echo Installing emscripten $(EMSCRIPTEN_VERSION)
-      sudo git reset HEAD --hard
-      sudo git pull
-      sudo ./emsdk update-tags
-      sudo ./emsdk install $(EMSCRIPTEN_VERSION)
-      sudo ./emsdk activate $(EMSCRIPTEN_VERSION)
-      sudo chmod -R 777 /emsdk
-      sudo chmod -R 777 ~/.emscripten_cache
-
+      cd ~
+      git clone https://github.com/emscripten-core/emsdk
+      cd emsdk
+      ./emsdk install sdk-1.39.11
+      ./emsdk activate sdk-1.39.11
+    
     displayName: 'Setup Emscripten'
 
   - bash: |
-      cd /emsdk
-      wget https://raw.githubusercontent.com/mono/mono/master/sdks/builds/emscripten-pr-8457.diff
-      wget https://raw.githubusercontent.com/mono/mono/master/sdks/builds/fix-emscripten-8511.diff
-      cd /emsdk/upstream/emscripten
-      patch -N -p1 < /emsdk/emscripten-pr-8457.diff
-      patch -N -p1 < /emsdk/fix-emscripten-8511.diff
-
+      cd ~/emsdk
+      wget https://raw.githubusercontent.com/mono/mono/27247739c68faee7b2a63ae805222d4b375d2161/sdks/builds/fix-emscripten-8511.diff
+      cd ~/emsdk/upstream/emscripten
+      patch -N -p1 < ~/emsdk/fix-emscripten-8511.diff
+    
     displayName: 'Apply Emscripten Mono patches'
 
-  - task: GitVersion@4
+  - task: UseDotNet@2
+    inputs:
+      packageType: runtime
+      version: 2.2.x
+      installationPath: /usr/share/dotnet
```

Comment:
We can remove that step if we use the `UseGitVersion@5`. Discovered that recently :)